### ISSUE TITLE
Tasks for Spring Boot 2.6.7 and 2.5.13

### ISSUE THREAD

**Issue by chenrujun** [Azure/azure-sdk-for-java#28498]
### Context

Spring Boot released [2.6.7](https://github.com/spring-projects/spring-boot/releases/tag/v2.6.7).

We should upgrade external dependencies' version align with [spring-boot-dependencies:2.6.7](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.7/spring-boot-dependencies-2.6.7.pom) managed versions. pring-boot-dependencies:2.6.7 will fix CVE-2022-22968.

### Goal

1. Make our libraries compatible with latest Spring Boot: 2.6.7 by making external dependencies' version align to spring-boot-dependencies:2.6.7 managed versions
2. Make our libraries compatible with latest Spring Boot: 2.5.13 by running unit test when set external dependencies' version align to Spring Boot: 2.5.13
3. Fix CVE-2022-22968

### Solution

Tasks are copied from this page: https://github.com/Azure/azure-sdk-for-java/wiki/Spring-Cloud-Azure-tasks-for-a-new-Spring-Boot-release

- Before azure-sdk-for-java in band release
  - [x] Create a new issue to track the following tasks. Sample: [#27863](https://github.com/Azure/azure-sdk-for-java/issues/27863).
  - [x] Read spring boot release notes. Sample: [Spring-Boot-2.6-Release-Notes](https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.6-Release-Notes). Write down items we(Spring Cloud Azure team members) should care about. Just write these items by adding a new comment in the GitHub issue created in previous step.
  - [x] Upgrade external dependencies' version according to spring-boot-dependencies and spring-cloud-dependencies. Refer to [related readme](https://github.com/Azure/azure-sdk-for-java/blob/aec4c6247ba7ba4de57dd866e3f5511ca4fbd387/sdk/spring/scripts/README.md). **NOTE**: This step should be finished before `Core code complete`. You can get more information about `Core code complete` in the Email with subject like `APRIL Release Kickoff`. https://github.com/Azure/azure-sdk-for-java/pull/28521

- After azure-sdk-for-java in band release.

  Update spring-boot and spring-cloud version in these places:

  - [x] [azure-spring-boot-samples](https://github.com/Azure-Samples/azure-spring-boot-samples)
  - [x] [Reference doc](https://github.com/microsoft/spring-cloud-azure/tree/docs/docs/src/main/asciidoc).
  - [ ] [MS docs](https://docs.microsoft.com/en-us/azure/developer/java/spring-framework/).
  - [x] [Wiki page: Spring Cloud Azure Timeline](https://github.com/Azure/azure-sdk-for-java/wiki/Spring-Cloud-Azure-Timeline)
  - [x] [Wiki page: Spring Versions Mapping](https://github.com/Azure/azure-sdk-for-java/wiki/Spring-Versions-Mapping)
  - [x] [Wiki page: Spring Support Multiple Versions](https://github.com/Azure/azure-sdk-for-java/wiki/Spring-Support-Multiple-Versions)

**Comment by Netyyyy** [comment]
close in #28521

**Comment by chenrujun** [comment]
@Netyyyy MS docs is not updated. For example, the screenshot in this page: https://docs.microsoft.com/en-us/azure/developer/java/spring-framework/spring-boot-starters-for-azure

Could you please update all contents related to spring-boot version in this PR: https://github.com/MicrosoftDocs/azure-dev-docs-pr/pull/3035

**Comment by Netyyyy** [comment]
> @Netyyyy Muyao Feng (CSI Interfusion Inc) Vendor MS docs is not updated. For example, the screenshot in this page: https://docs.microsoft.com/en-us/azure/developer/java/spring-framework/spring-boot-starters-for-azure
> 
> Could you please update all contents related to spring-boot version in this PR: [MicrosoftDocs/azure-dev-docs-pr#3035](https://github.com/MicrosoftDocs/azure-dev-docs-pr/pull/3035)

will track in https://github.com/Azure/azure-sdk-for-java/issues/29658
